### PR TITLE
JavaScript capstone project - Your API-based webapp

### PR DESCRIPTION
**Pull request by astroboyReloaded**
## For this capstone project we did the following: 📑 

- Cose the [TVmaze API](https://www.tvmaze.com/api). 🔗 
- Set up project with `linters`, `webpack`, and `jest`. 🛠️ 
- Created the `HTML` file and wrote the **shared code** (header and footer). 🧱 
- Created our [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) id. 🔑 
- Make a **GET** request function to the **TVmaze API** to fetch the shows data. ☎️ 
- Make a **POST** request function to the **Involvement API** to send and record **likes** for each show. ❤️ 
- Make a **GET** request function to the **Involvement API** to fetch the comments for each show. 📞 
- Make a **POST** request function to the **Involvement API** to send and record **comments** for each show. 💬 
- Write counter functions for likes and comments. ➕ 
- Write tests for counter functions using **Jest**. ✔️

### CODE REVIEW

File: src/modules/likes.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Async function to get likes for a show
+export const getLikes = async () => {
+  await fetch(
+    'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/SzueLQl3FaQn1wJjDbsx/likes',
+  )
+    .then((response) => response.json())
+    // eslint-disable-next-line no-use-before-define
```
Comment (rustam-tolipov) [inline src/modules/likes.js]: - [ ] To make your code clearn, kindly remove `eslint-disabler` from your project! 

> Suggestion: At that point. You can fix the issue by putting getLikes function after renderLikes. Then your eslint issue will be fixed

File: src/modules/likes.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    .then((data) => renderLikes(data));
+};
+const renderLikes = (likes) => {
+  const likesContainers = document.querySelectorAll('.likesContainer');
+  likesContainers.forEach((container) => {
+    // eslint-disable-next-line array-callback-return
```
Comment (rustam-tolipov) [inline src/modules/likes.js]: - [ ] In this case your filter is looking for item to return. You can put likes to return. Kindly remove the eslint-disabler

File: src/index.js [hunk 3]
```diff
@@ -1,12 +1,61 @@
-import _ from 'lodash';
-import './style.css'
+/* eslint-disable no-plusplus */
+/* eslint-disable no-use-before-define */
```
Comment (rustam-tolipov) [inline src/index.js]: - [ ] The last one please remove these disablers as well.


You also can these disablers as well. By running:
```
npx eslint . --fix
```

in your terminal. This command fix both above eslint.. issue.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Great implementations on tests_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linter are passing
✅ Implemented reviewer's suggestions properly
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**